docs/source/usersguide/materials.rst

```diff
@@ -101,6 +101,42 @@ you would need to add hydrogen and oxygen to a material and then assign the
 
 .. _usersguide_naming:
 
+------------------
+Adding NCrystal materials
+------------------
+
+Additional support for thermal scattering can be added by using NCrystal_.
+The :meth:`Material.from_ncrystal` class method generates a material object from
+an `NCrystal configuration string <https://github.com/mctools/ncrystal/wiki/Using-NCrystal#uniform-material-configuration-syntax>`_.
+Temperature, material composition and density are passed from the configuration string
+and the `NCMAT file <https://github.com/mctools/ncrystal/wiki/NCMAT-format>`_
+that define the material, e.g:
+
+::
+   
+   mat = openmc.Material.from_ncrystal('Al_sg225.ncmat')
```

Comment:
should we perhaps use `'Al_sg225.ncmat;temp=300K'` since many users most likely want to know how to set the temperature?